slack-im.c

```diff
@@ -170,7 +170,8 @@ int slack_send_im(PurpleConnection *gc, const char *who, const char *msg, Purple
 	send->flags = flags;
 
 	if (!*user->im)
-		slack_api_post(sa, send_im_open_cb, send, "im.open", "user", user->object.id, "return_im", "true", NULL);
+		//slack_api_post(sa, send_im_open_cb, send, "im.open", "user", user->object.id, "return_im", "true", NULL);
```

Comment:
Any reason to keep the old call commented out and not removed entirely?

Reply:
No, no reason. I thought I had removed it, but apparently not!

Reply:
New version forced up. I don't like silly fixups ;)